templates/index.html

```diff
@@ -193,7 +197,12 @@ <h4 class="{{#experiment}}amp-experiment-medium{{/experiment}}">
               {{title}}
             </h4> 
             {{#description}} <p class="description">{{{ description }}}</p> {{/description}}
-            <a class="button" href="{{url}}">View Sample</a>
+            <div class="buttons">
+              <a class="button" href="{{url}}">View Sample</a>
+              {{#metadata.preview}}
+              <a class="button" href="{{urlPreview}}">View Preview</a>
```

Comment:
Should we use "show" instead of "view" to be consistent to what we are already using on the top right of the samples with preview?